mantis-shaded-wrapper/build.gradle

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Please add a README in this directory to clearly state this sub-project is only getting intellij to work. And include the link to the github issue where this workaround is discussed.

Reply:
Will do